packages/build/src/core/config.js

```diff
@@ -34,7 +34,7 @@ const isDefined = function(key, value) {
 // Default values of CLI flags
 const DEFAULT_FLAGS = {
   token: NETLIFY_AUTH_TOKEN,
-  context: CONTEXT || 'production',
+  context: CONTEXT || 'local',
```

Comment:
Local value is going to break people using config.context override values if they run locally. This needs to be an existing value

Reply:
See https://docs.netlify.com/configure-builds/file-based-configuration/#sample-file for more info on context

Reply:
Added as #792.